### PR TITLE
Week 5 day 1

### PR DESCRIPTION
**Pull request by sgharms**
Steven review notes

### CODE REVIEW

File: public/javascripts/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var calculator = {add: function(x, y) {
+                    return (x + y);
```
Comment (sgharms) [inline public/javascripts/main.js:2]: Dang girl, your indentation is bizonkers.  Please figure out how to get sublime to indent with 2 spaces.

File: public/javascripts/main.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+                  },
+                  subtract: function(x, y) {
+                    return (x - y);
+                  },
+                  multiply: function(x, y) {
+                    return (x * y);
+                  },
+                  divide: function(x, y) {
+                    return (x / y);
```
Comment (sgharms) [inline public/javascripts/main.js:11]: what if y is 0 ?

File: spec/javascripts/javascript_spec.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    var num1 = 12, num2 = 3;
+    expect(calculator.subtract(num1, num2)).toEqual(9);
+  });
+
+  it("should multiply 2 numbers", function() {
+    var num1 = 12, num2 = 3;
+    expect(calculator.multiply(num1, num2)).toEqual(36);
+  });
+
+  it("should divide 2 numbers", function() {
```
Comment (sgharms) [inline spec/javascripts/javascript_spec.js:44]: Good tests.  Remember, writing tests that catch edge cases (div by 0) is a vital part of writing tests.
